src/app/home/layouts/income-form/income-form.component.html

```diff
@@ -6,7 +6,7 @@ <h2 class="header">Set income</h2>
       (swipeup)="toggleActiveOnSwipeUp()"
       (swipedown)="toggleActiveOnSwipeDown()">
 
-  <ng-container *ngFor="let i of incomeNumber | fillArray" [ngModelGroup]="i">
+  <div *ngFor="let i of incomeNumber | fillArray" [ngModelGroup]="i" class="form-container">
```

Comment:
"fillArray" is not very accurate name, as it does not really tell what that pipe does. From the name, it might suggest that the result will be 'fill array with incomeNumber the unknown number of times', and it rather fills it 'incomeNumber times with whatever item'.  Maybe try naming it 'times', 'numberSequence', 'series', or something like that.